src/contacts/processors/impl/AddNewPersonActionProcessor.java

```diff
@@ -0,0 +1,44 @@
+package contacts.processors.impl;
+
+import contacts.ConsoleReader;
+import contacts.model.Person;
+import contacts.model.Record;
+import contacts.processors.IRecordActionProcessor;
+import contacts.service.IRecordService;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+public class AddNewPersonActionProcessor implements IRecordActionProcessor {
+
+    private final IRecordService recordService;
+
+    public AddNewPersonActionProcessor(IRecordService personService) {
+        this.recordService = personService;
+    }
+
+    @Override
+    public void doAction() throws IOException {
+        String name = ConsoleReader.getStringFromConsole("Enter the name of the person:");
+        String surname = ConsoleReader.getStringFromConsole("Enter the surname of the person:");
+        String birthDate = ConsoleReader.getStringFromConsole("Enter the birth date:");
+        String gender = ConsoleReader.getStringFromConsole("Enter the gender (M, F):");
+        System.out.println("Bad gender!");
+        String number = ConsoleReader.getStringFromConsole("Enter the phone number:");
+
+        Record record = new Person(name, surname, "[no data]", "[no data]");
+        record.setPerson(true);
+        record.setPhoneNumber(number);
```

Comment:
why don't you validate phone number?

Reply:
This is a requirement of the program for passing the tests